### ISSUE TITLE
Box.once should wait before the server enters RW mode

### ISSUE THREAD

**Issue by mejedi** [tarantool/tarantool#1075]
```lua
local function bootstrap()
    s = box.schema.space.create('tester')
end

box.cfg{listen=3000, replication_source='sh2.tarantool.org:3000'}

box.once('bootstrap', bootstrap) -- complains about RO mode
```

**Comment by kostja** [comment]
This should be fixed by now. Please add a test.

**Comment by rtsisyk** [comment]
This bug is a duplicate of #999.
Kostja'a comment was about #1101.

**Comment by rtsisyk** [comment]
Discussed with @kostja.  Nov 13, 2015 comment is a mistake. This ticket is similar to #999, but for 1.6.x and should be fixed in the different way. box.cfg{}/box_init() should wait for LSN returned by SUBSCRIBE (see #691) before exiting to the caller.

Unfortunately, required synchronization primitives for applier were pushed to 1.7 and won't be ported back to 1.6.8. Please try find a primitive workaround to wait for LSN in box_init().

**Comment by rtsisyk** [comment]
Fixed in 1.6.8-396-g32ff24d
